Umbraco-Cloud/Set-Up/Working-Locally/index.md

```diff
@@ -5,80 +5,216 @@ versionTo: 10.0.0
 
 # Working with a Local Clone of an Umbraco Cloud Site
 
-To work with a local copy of your site, you'll need to use Windows and have a local web server installed (like WebMatrix / IIS). If you're not using Windows, you can still work with your site's files (Templates, Css, JavaScript, etc...) but you'll need to deploy these to your development site before you can "see" your updates - head over to our chapter on [Working with UaaS Cli](../Working-With-UaaS-Cli/) for more information on this.
+To work with a local copy of your site, you'll need to use Windows with a local web server installed (like WebMatrix/IIS). If you're using Mac, see the [Working with Mac](../Working-With-UaaS-Cli/) article. In this article, you will find:
+
+- [Video Tutorial](#video-tutorial)
+- [Cloning an Umbraco Cloud Project](#cloning-an-umbraco-cloud-project)
+- [Running the Site Locally](#running-the-site-locally)
+- [Working with Visual Studio](#working-with-visual-studio)
+    - [Adding a Solution File to your Cloud Project](#adding-a-solution-file-to-your-cloud-project)
+    - [Adding Additional Projects to your Solution](#adding-additional-projects-to-your-solution)
+- [Renaming the Project Files and Folders](#renaming-the-project-files-and-folders)
 
 ## Video Tutorial
 
 <iframe width="800" height="450" src="https://www.youtube.com/embed/e3spd6Nqrf8?rel=0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Cloning the Umbraco Project
+## Cloning an Umbraco Cloud Project
+
+To clone an Umbraco cloud project, follow these steps:
+
+1. On the Umbraco Cloud portal, **Go to the project** you wish to clone.
+2. Click on the arrow next to the **Development** environment and select **Clone project**.
+
+   ![Clone project option](images/clone-project-option.png)
+3. **Copy** the clone URL to copy the development environment's git repository endpoint.
+
+    ![Copy the clone URL](images/copy-clone-url.png)
+4. Go to the location on your local machine where you wish to clone the project.
+5. Clone the repository using your favorite Git client. For example: [Fork](https://git-fork.com/), [SourceTree](https://www.sourcetreeapp.com/), [Git Extensions](https://gitextensions.github.io), [GitKraken](https://www.gitkraken.com/), etc. For this article, I'll use **Git Bash**.
```

Comment:
How about moving the section about the git clients to a "Tip" below or above the steps?
Or perhaps mention it in a "Prerequisites" or "Tools" section at the top of the article, like we do in this article: https://our.umbraco.com/documentation/Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/#2-tools

Also, I've suggested to merged steps 4 and 5, to simplify the process a bit. Hope it still makes sense!

```suggestion
5. Use your favorite Git client to clone down the project. In this guide, we will continue to use **Git Bash**.
```